Android build failed

Using expo 54 with crisp     "crisp-sdk-react-native": "^0.2.0",








> Task :app:checkDebugDuplicateClasses FAILED


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class org.commonmark.Extension found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.BlockContent found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.BlockContinueImpl found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.BlockQuoteParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.BlockQuoteParser$Factory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.BlockStartImpl found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.Bracket found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.Delimiter found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.DocumentBlockParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.DocumentParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.DocumentParser$MatchedBlockParserImpl found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.FencedCodeBlockParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.FencedCodeBlockParser$Factory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.HeadingParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.HeadingParser$Factory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.HtmlBlockParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.HtmlBlockParser$1 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.HtmlBlockParser$Factory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.IndentedCodeBlockParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.IndentedCodeBlockParser$Factory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.InlineParserContextImpl found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.InlineParserImpl found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.InlineParserImpl$DelimiterData found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.LinkReferenceDefinitionParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.LinkReferenceDefinitionParser$1 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.LinkReferenceDefinitionParser$State found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.ListBlockParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.ListBlockParser$Factory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.ListBlockParser$ListData found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.ListBlockParser$ListMarkerData found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.ListItemParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.ParagraphParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.StaggeredDelimiterProcessor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.ThematicBreakParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.ThematicBreakParser$Factory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.inline.AsteriskDelimiterProcessor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.inline.EmphasisDelimiterProcessor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.inline.UnderscoreDelimiterProcessor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.renderer.NodeRendererMap found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.renderer.text.BulletListHolder found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.renderer.text.ListHolder found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.renderer.text.OrderedListHolder found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.util.Escaping found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.util.Escaping$1 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.util.Escaping$2 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.util.Escaping$Replacer found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.util.Html5Entities found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.util.LinkScanner found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.internal.util.Parsing found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.AbstractVisitor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Block found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.BlockQuote found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.BulletList found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Code found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.CustomBlock found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.CustomNode found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Delimited found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Document found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Emphasis found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.FencedCodeBlock found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.HardLineBreak found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Heading found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.HtmlBlock found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.HtmlInline found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Image found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.IndentedCodeBlock found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Link found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.LinkReferenceDefinition found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.ListBlock found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.ListItem found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Node found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.OrderedList found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Paragraph found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.SoftLineBreak found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.StrongEmphasis found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Text found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.ThematicBreak found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.node.Visitor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.InlineParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.InlineParserContext found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.InlineParserFactory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.Parser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.Parser$1 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.Parser$Builder found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.Parser$Builder$1 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.Parser$ParserExtension found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.PostProcessor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.block.AbstractBlockParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.block.AbstractBlockParserFactory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.block.BlockContinue found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.block.BlockParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.block.BlockParserFactory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.block.BlockStart found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.block.MatchedBlockParser found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.block.ParserState found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.delimiter.DelimiterProcessor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.parser.delimiter.DelimiterRun found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.NodeRenderer found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.Renderer found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.AttributeProvider found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.AttributeProviderContext found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.AttributeProviderFactory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.CoreHtmlNodeRenderer found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.CoreHtmlNodeRenderer$1 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.CoreHtmlNodeRenderer$AltTextVisitor found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.HtmlNodeRendererContext found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.HtmlNodeRendererFactory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.HtmlRenderer found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.HtmlRenderer$1 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.HtmlRenderer$Builder found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.HtmlRenderer$HtmlRendererExtension found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.HtmlRenderer$RendererContext found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.html.HtmlWriter found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.CoreTextContentNodeRenderer found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.TextContentNodeRendererContext found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.TextContentNodeRendererFactory found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.TextContentRenderer found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.TextContentRenderer$1 found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.TextContentRenderer$Builder found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.TextContentRenderer$RendererContext found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.TextContentRenderer$TextContentRendererExtension found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)
     Duplicate class org.commonmark.renderer.text.TextContentWriter found in modules commonmark-0.13.0.jar -> commonmark-0.13.0 (com.atlassian.commonmark:commonmark:0.13.0) and commonmark-0.21.0.jar -> commonmark-0.21.0 (org.commonmark:commonmark:0.21.0)

     Learn how to fix dependency resolution errors at https://d.android.com/r/tools/classpath-sync-errors

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.14.3/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

--- a/.changeset/fix-android-commonmark-conflict.md
+++ b/.changeset/fix-android-commonmark-conflict.md
@@ -1,0 +1,5 @@
+---
+"crisp-sdk-react-native": patch
+---
+
+Fix Android dependency resolution by excluding the CommonMark version pulled by the Crisp Android SDK and pinning a compatible CommonMark dependency.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -14,5 +14,8 @@ android {
 }
 
 dependencies {
-  implementation "im.crisp:crisp-sdk:2.0.21"
+  implementation("im.crisp:crisp-sdk:2.0.21") {
+    exclude group: "com.atlassian.commonmark", module: "commonmark"
+  }
+  implementation "org.commonmark:commonmark:0.21.0"
 }

--- a/plugin/src/withAndroid.ts
+++ b/plugin/src/withAndroid.ts
@@ -15,6 +15,44 @@ type AndroidNotificationConfig = {
   mode: NotificationMode;
 };
 
+const CRISP_ANDROID_SDK_DEPENDENCY = `implementation('im.crisp:crisp-sdk:2.0.21') {
+        exclude group: 'com.atlassian.commonmark', module: 'commonmark'
+    }`;
+const COMMONMARK_ANDROID_DEPENDENCY = "implementation 'org.commonmark:commonmark:0.21.0'";
+const CRISP_COMMONMARK_EXCLUSION =
+  "exclude group: 'com.atlassian.commonmark', module: 'commonmark'";
+
+function addCrispAndroidSdkDependency(contents: string): string {
+  if (!contents.includes("im.crisp:crisp-sdk")) {
+    return contents.replace(
+      /dependencies\s*\{/,
+      `dependencies {
+    ${CRISP_ANDROID_SDK_DEPENDENCY}`,
+    );
+  }
+
+  if (contents.includes(CRISP_COMMONMARK_EXCLUSION)) {
+    return contents;
+  }
+
+  return contents.replace(
+    /implementation\s+['"]im\.crisp:crisp-sdk:2\.0\.21['"]/,
+    CRISP_ANDROID_SDK_DEPENDENCY,
+  );
+}
+
+function addCommonmarkAndroidDependency(contents: string): string {
+  if (contents.includes("org.commonmark:commonmark")) {
+    return contents;
+  }
+
+  return contents.replace(
+    /dependencies\s*\{/,
+    `dependencies {
+    ${COMMONMARK_ANDROID_DEPENDENCY}`,
+  );
+}
+
 export const withAndroidNotifications: ConfigPlugin<[string, NotificationMode]> = (
   config,
   [websiteId, mode],
@@ -284,13 +322,8 @@ const withAndroidCoexistence: ConfigPlugin<AndroidNotificationConfig> = (config,
     implementation 'com.google.firebase:firebase-messaging'`,
       );
     }
-    if (!config.modResults.contents.includes("im.crisp:crisp-sdk")) {
-      config.modResults.contents = config.modResults.contents.replace(
-        /dependencies\s*\{/,
-        `dependencies {
-    implementation 'im.crisp:crisp-sdk:2.0.21'`,
-      );
-    }
+    config.modResults.contents = addCrispAndroidSdkDependency(config.modResults.contents);
+    config.modResults.contents = addCommonmarkAndroidDependency(config.modResults.contents);
     return config;
   });
 

